src/tests/ftest/object/obj_update_bad_param.py

```diff
@@ -39,6 +39,23 @@ class ObjUpdateBadParam(TestWithServers):
     def setUp(self):
         super(ObjUpdateBadParam, self).setUp()
         self.plog = logging.getLogger("progress")
+        try:
+            self.prepare_pool()
+            # create a container
+            self.container = DaosContainer(self.context)
+            self.container.create(self.pool.pool.handle)
+            self.plog.info("Container %s created.", self.container.get_uuid_str())
+            # now open it
+            self.container.open()
+        except DaosApiError as excep:
+            print(excep)
+            print(traceback.format_exc())
+            self.fail("Test failed during setup .\n")
+
+    def tearDown(self):
+        if self.container:
+            self.container.close()
+            self.container.destroy()
```

Comment:
Don't think this is needed. 
tearDown() in apricot should already be doing that for any container if any.

Reply:
Done.